lib/routes/sara/index.ts

```diff
@@ -0,0 +1,66 @@
+import { Route, Data } from '@/types';
+import cache from '@/utils/cache';
+import { load } from 'cheerio';
+import { ofetch } from 'ofetch';
+
+const typeMap = {
+    dynamic: '协会动态',
+    announcement: '通知公告',
+    industry: '行业动态',
+};
+
+export const route: Route = {
+    path: '/:type',
+    categories: ['government'],
+    example: '/sara/announcement',
+    parameters: { type: 'dynamic | announcement | industry' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    description: `| 协会动态 | 通知公告 |行业动态 |
+    | -------- | ------------ | -------- |
+    | dynamic | announcement | industry |`,
+
+    name: '新闻资讯',
+    maintainers: ['HChenZi'],
+    handler: async (ctx) => {
+        const baseUrl = 'http://www.sara.org.cn';
+        const type = ctx.req.param('type');
+
+        const url = `${baseUrl}/news/${type}.htm`;
+        const response = await ofetch(url);
+        const $ = load(response);
+        const list = $('.newsItem_total > dd')
+            .toArray()
+            .map((item) => {
+                const a = $(item).find('a').first();
+                return {
+                    link: `${baseUrl}${a.attr('href')}`,
+                    title: a.attr('title'),
+                };
+            });
+        const items = (await Promise.all(list.map(getFeedItem))) as Data[];
```

Comment:
It should be `DataItem[]`. 
`Data` is the type of returned object of `handler`.

Reply:
👌👌